hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/checker/ValidFieldsChecker.groovy

```diff
@@ -0,0 +1,54 @@
+package com.linkedin.gradle.hadoopdsl.checker;
+
+import com.linkedin.gradle.hadoopdsl.BaseStaticChecker;
+import com.linkedin.gradle.hadoopdsl.job.SparkJob;
+import org.gradle.api.Project;
+
+class ValidFieldsChecker extends BaseStaticChecker {
```

Comment:
Could you add a comment to the class description indicating that the purpose of this checker is to inspect for valid field values (whether or not the fields are required), rather than just the presence or absence of required fields?